### ISSUE TITLE
JBQ Coach's App

### ISSUE THREAD

**Issue by ChristopherMank** [biblequiz/BibleQuiz.com#222]
It looks like the Coach's App published to BibleQuiz.com is for TBQ only. Are there plans to expand this to JBQ and/or import the JBQ Coach's Assistant app here?

**Comment by michsco** [comment]
@ChristopherMank -- my apologies for the delay in response. My notification settings were incorrect.

@kevintgallo can provide a little more context, but the Coach's App isn't just for TBQ. I believe Kevin uses it for JBQ coaching as well.

**Comment by kevintgallo** [comment]
I am not sure why you think it is for TBQ - but it was originally designed for JBQ.  Is there some documentation or branding that is unclear or misleading?

For downloaded meets, it will use the competition type set in the Event.  Otherwise, you pick the type of Event you want to create when creating the event itself.

I have created a Google Group (bqcoachapp@googlegroups.com) you can join to get information and to share feedback on.

Several coaches used this at our last JBQ meet and we have a few bugs we need to fix but otherwise it went really well.  If you need some info on how to set it up or use it, let me know.  I did a video recording of training for our league coaches but I have not uploaded to YouTube yet.

**Comment by kevintgallo** [comment]
I updated the description on the website to list JBQ and TBQ explicitly.

**Comment by ChristopherMank** [comment]
@kevintgallo Thanks! I guess I didn't look hard enough as I only saw the Romans/James Scripture portion and Select Season settings were also pointing to TBQ. I just requested to join the Google group as I'd love to hear more and give feedback. We don't use electronic scoring in our league meets yet so hoping the workflow to simply add an ad-hoc match would be easier or perhaps I'm again just not understanding it fully. Thanks!
